{"name": "test_add[3-5-8]", "status": "failed", "statusDetails": {"message": "AssertionError: assert -2 == 8\n +  where -2 = subtract_function(3, 5)", "trace": "a = 3, b = 5, expected = 8\n\n    @pytest.mark.parametrize(\"a,b,expected\",\n                            [(3,5,8),\n                            (-1,-2,-3),\n                            (100,200,300),\n                            ])\n    def test_add(a,b,expected):\n        assert add_function(a,b) == expected\n>       assert subtract_function(a,b) == expected\nE       assert -2 == 8\nE        +  where -2 = subtract_function(3, 5)\n\ntest_作业计算器.py:20: AssertionError"}, "parameters": [{"name": "a", "value": "3"}, {"name": "b", "value": "5"}, {"name": "expected", "value": "8"}], "start": 1701408934314, "stop": 1701408934314, "uuid": "7407ee49-bc1d-4425-92e0-52924880882b", "historyId": "bae6ab89f5976bb487e41676476946f8", "testCaseId": "c8cde32a51c929970e763894868faaa8", "fullName": "test_作业计算器#test_add", "labels": [{"name": "suite", "value": "test_作业计算器"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "17248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_作业计算器"}]}